diff -urN kcmsystemd-0.6.0/ui/kcmsystemd.ui kcmsystemd-0.6.0-patched/ui/kcmsystemd.ui
--- kcmsystemd-0.6.0/ui/kcmsystemd.ui	2014-03-28 09:04:10.000000000 +1100
+++ kcmsystemd-0.6.0-patched/ui/kcmsystemd.ui	2014-03-29 20:43:09.663945097 +1100
@@ -200,7 +200,7 @@
                 <bool>false</bool>
                </property>
                <property name="toolTip">
-                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Starts the selected unit.&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Note: &lt;/span&gt;This does not enable autostart of unit. For this purpose right-click unit and select &amp;quot;Enable unit&amp;quot;.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
+                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Starts the selected unit.&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Note: &lt;/span&gt;This does not enable autostart of unit. For this purpose right-click unit and select &quot;Enable unit&quot;.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="text">
                 <string>Start</string>
@@ -213,7 +213,7 @@
                 <bool>false</bool>
                </property>
                <property name="toolTip">
-                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Stops the selected unit.&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Note: &lt;/span&gt;This does not disable autostart of unit. For this purpose right-click unit and select &amp;quot;Disable unit&amp;quot;.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
+                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Stops the selected unit.&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Note: &lt;/span&gt;This does not disable autostart of unit. For this purpose right-click unit and select &quot;Disable unit&quot;.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="text">
                 <string>Stop</string>
@@ -1627,7 +1627,7 @@
               <item row="6" column="1">
                <widget class="QSpinBox" name="spnInhibitDelayMaxSec">
                 <property name="toolTip">
-                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The maximum time a system shutdown or sleep request is delayed due to an inhibitor lock of type &amp;quot;delay&amp;quot; being active.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
+                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The maximum time a system shutdown or sleep request is delayed due to an inhibitor lock of type &quot;delay&quot; being active.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                 </property>
                 <property name="suffix">
                  <string/>
@@ -1910,7 +1910,7 @@
               <item row="0" column="1">
                <widget class="QLineEdit" name="leControllers">
                 <property name="toolTip">
-                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Default control group hierarchies users logging in are added to, in addition to the &amp;quot;name=systemd&amp;quot; hierarchy. Comma-separated list of controllers.&lt;/p&gt;&lt;p&gt;This option was removed in systemd 207.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
+                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Default control group hierarchies users logging in are added to, in addition to the quot;name=systemdquot; hierarchy. Comma-separated list of controllers.&lt;/p&gt;&lt;p&gt;This option was removed in systemd 207.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                 </property>
                </widget>
               </item>
