diff -urN kcmsystemd-0.5.0/CMakeLists.txt kcmsystemd-0.5.0-patched/CMakeLists.txt
--- kcmsystemd-0.5.0/CMakeLists.txt	2013-12-30 10:28:11.000000000 +1100
+++ kcmsystemd-0.5.0-patched/CMakeLists.txt	2014-01-13 13:21:14.322398231 +1100
@@ -17,7 +17,7 @@
 
 set(kcmsystemd_INCLUDE_DIRS ${KDE4_INCLUDES})
 
-set(CMAKE_CXX_FLAGS "-fexceptions")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fexceptions")
 
 configure_file(config.h.cmake ${CMAKE_CURRENT_BINARY_DIR}/src/config.h)
 
